src/scripts/modules/media/editbar/block-publish/modals/block-publish-template.html

```diff
@@ -0,0 +1,18 @@
+<div id="reject-publish-modal" class="modal fade">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <form class="form-horizontal">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+          <h4 class="modal-title">You Cannot Publish</h4>
+        </div>
+        <div class="modal-body">
+          You cannot publish because of the following reasons: {{publishBlockers}}
```

Comment:
You don't need to write a handlebars helper for this.  The `templateHelpers` property exists to allow you to transform/append any data before passing it to the template.